modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -4830,6 +4840,89 @@ type SpectroscopyResult {
   results: [Result!]!
 }
 
+"""Exposure calculation result types"""
+enum ExposureTimeResultType {
+  SUCCESS
+
+  SOURCE_TOO_BRIGHT
+
+  CALCULATION_ERROR
+}
+
+# Possible exposure time itc result
+interface ExposureTimeResult {
+  # Result type
+  resultType: ExposureTimeResultType!
```

Comment:
Are the subtypes and the enum both necessary? Clients can query what of the subtypes is returned by asking for `__typename`.

Reply:
ah you can tell when people know their stuff 😄